gui.h

```diff
@@ -3856,6 +4003,17 @@ const b32 g_gui_npt_chars_numeric[128] = {
 	0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
 };
 
+const b32 g_gui_npt_chars_hex[128] = {
+	0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
+	0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
+	0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
+	1,1,1,1,1,1,1,1,1,1,0,0,0,0,0,0,
+	0,1,1,1,1,1,1,0,0,0,0,0,0,0,0,0,
+	0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,0,
+	0,1,1,1,1,1,1,0,0,0,0,0,0,0,0,0,
+	0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,0,
+};
+
```

Comment:
It's an ascii character filter.  1=allowed, 0=disallowed.  So you can't type a 'z' into a input box for hex values.  I like this filtering, but I have no idea if it will work if/when Unicode support is added.